.github/workflows/download_pipeline.yml

```diff
@@ -69,4 +69,4 @@ jobs:
         env:
           NXF_SINGULARITY_CACHEDIR: ./
           NXF_SINGULARITY_HOME_MOUNT: true
-        run: nextflow run ./${{ env.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< ${{ env.REPO_BRANCH }}) -stub -profile test,singularity --outdir ./results
+        run: nextflow run ./${{ env.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< ${{ env.REPO_BRANCH }}) -stub -profile test,singularity --outdir ./results --input https://raw.githubusercontent.com/nf-core/test-datasets/readsimulator/samplesheet/test_samplesheet.csv
```

Comment:
why don't you provide the input in the test config? easier to keep it in sync

Reply:
The problem is that I did provide the input in the test config. I've put it in conf/test.config. So maybe this is a case of a misleading error message because PR #41 has a failing test with the error "Missing required parameter: --input"

Reply:
Maybe I should remove the -stub option instead. [This](https://nfcore.slack.com/archives/CE6SDBX2A/p1711612916626719) discussion on Slack makes it seem like the -stub option could be the issue.